src/views/withNavigation.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import React from 'react';
+
+import { ContextWithNavigation } from '../TypeDefinition';
+
+export default function (Component: ReactClass<T>) {
```

Comment:
Can you also set displayName ? https://github.com/react-community/react-navigation/blob/master/src/withCachedChildNavigation.js#L25